src/cmd.rs

```diff
@@ -167,11 +168,33 @@ pub async fn connect_ovpn(
     let mut lines = buf.lines();
 
     let mut next = lines.next_line().await;
+    let mut dns_server = String::new();
+    let mut tun_device = String::new();
+    let mut resolvectl_run = false;
 
     loop {
         if let Ok(ref line) = next {
             if let Some(line) = line {
                 log.append_process(pid, line.as_str());
+                let dns_server_some = parse_dns(line.to_string());
+                if dns_server_some.is_some() {
+                    dns_server = dns_server_some.as_deref().unwrap().to_string();
+                    println!("DNS Server is {:?}", dns_server);
```

Comment:
Use this instead, so the user can see in the UI:
```suggestion
                    log.append_process(pid, format!("DNS Server is {:?}", dns_server).as_str());
```

If I'm not mistaken, there are just a few places were `println!` are used, but they either have a corresponding log call somewhere else (because this codebase is a mess), or they happen within the http server.